### ISSUE TITLE
Unable to use a path different than /

### ISSUE THREAD

**Issue by kamzata** [huggingface/Mongoku#22]
When I run Mongoku on _https://mydomain.com/_ it works. When I try to set it to run on (e.g.) _https://mydomain.com/mongo/_ it do not work. It shows a blank page. Inspecting it I could find out that .css and .js resources are not available because it try to load it on root absolute path (_https://mydomain.com/_).

**Comment by n1t0** [comment]
Indeed, you need to specify the path during the build phase using `—baseHref=“/mongo”`

**Comment by kamzata** [comment]
> Indeed, you need to specify the path during the build phase using `—baseHref=“/mongo”`

Thanks, I installed it using _npm_. Is there a way to set the path anyway?

**Comment by ddempsey3** [comment]
Would be ideal to pass in the path though environment variables, thanks for the great work on this UI!

**Comment by Serrvosky** [comment]
I'm facing the same problem
